docs/source/comms.rst

```diff
@@ -7,8 +7,10 @@ update widget state.
 
 A comm consists of a pair of objects, in the kernel and the frontend, with an
 automatically assigned unique ID. When one side sends a message, a callback on
-the other side is triggered with that message data. Either side can open or
-close the comm.
+the other side is triggered with that message data. For example, the kernel can
+send a message to the frontend which triggers a callback by the frontend to
+the kernel with the message data. Either side, the frontend or kernel, can
```

Comment:
This doesn't quite match my understanding. If there is a message from the kernel to the frontend, the callback is in the frontend. Callbacks are just function calls, so they're not 'to' another part.